### PR TITLE
removed old version number and replaced repo by 'current' version

### PR DESCRIPTION
**Pull request by keinhaar**
Also moved sponsors to the bottom of the page, because normal visitors will be more interested in installing the current version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,18 +42,11 @@ The Eclipse repositories for this plugin.
 Release update site.
 
 * [Eclipse Marketplace entry for the GWT Plugin](https://marketplace.eclipse.org/content/gwt-plugin)
-* [Update sites for the GWT Plugin](https://plugins.gwtproject.org/eclipse/gwt-eclipse-plugin/)
+* [Update sites for the GWT Plugin](https://gwt-plugins.github.io/documentation/repo/site/current/)
```
Comment (protoism) [inline README.md:45]: I'm really not sure that github a pages is a good place for storing eclipse plugins

File: README.md [hunk 2]
```diff
@@ -100,7 +82,7 @@ Creating a release requires signing artifacts. Set the following environment var
 ### Deploy
 Releases are uploaded as zips to the [release](https://github.com/gwt-plugins/gwt-eclipse-plugin/releases/)
```
Comment (protoism) [inline README.md:83]: I don't follow.
What files are on plugins.gwtproject.org/eclipse/gwt-eclipse-plugin?
What files are on gwt-plugins.github.io?

**Comment by protoism** [comment]
This PR is not complete, I believe... as the google pages hosting idea has been abandoned

**Comment by keinhaar** [comment]
> This PR is not complete, I believe... as the google pages hosting idea has been abandoned

You're right, i need to fix that.

**Pull request merged**